src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    public Duke() throws FileNotFoundException {
+        ui = new Ui();
+        this.storage = new Storage("../duke.txt");
+        this.tasks = new TaskList(storage.load());
+    }
+
+    public void run() throws DukeException, FileNotFoundException{
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> store = this.tasks.list;
+
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
+        try {
```

Comment:
You can consider extracting more of the codes in this long block to your different classes, such as having the Ui class to handle the printing, etc

Reply:
You can also abstract this block to the Parser class to have it do the parsing instead of having it in the main class